garak/generators/openai.py

```diff
@@ -145,7 +134,6 @@ def __init__(self, name, generations=10):
         (
             openai.RateLimitError,
             openai.InternalServerError,
-            openai.APIError,
```

Comment:
Removing the general error seems prudent given testing that shows swallows `4xx` responses and would cause backoff to trigger more than it should.